applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -0,0 +1,30 @@
+[id="configuring-custom-helm-chart-repositories"]
+
+= Configuring custom Helm chart repositories
+include::modules/common-attributes.adoc[]
+:context: configuring-custom-helm-chart-repositories
+
+toc::[]
+
+You can install Helm charts on an {product-title} cluster using the CLI. The *Developer Catalog*, in the *Developer* perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the Red Hat Helm chart repository. For a list of the charts see link:https://redhat-developer.github.io/redhat-helm-charts/index.yaml[the Red Hat `Helm index` file].
```

Comment:
Change the name too please from "Red Hat Helm chart repository" to "OpenShift Helm Chart repository". Also not sure it flows well. We start by saying that we can install helm charts using the cli and then jump to the developer perspective in the console. Those are two different experiences. On from the CLI and the other from the Console. Make sure they are clearly different paths.

Reply:
Changing to:

You can install Helm charts on an OpenShift Container Platform cluster using the following methods:

- Using the CLI 
- Using the **Developer** perspective of the web console.

The **Developer Catalog**, in the **Developer** perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the Red Hat OpenShift Helm chart repository. For a list of the charts see link:https://charts.openshift.io/index.yaml[the Red Hat `Helm index` file].